include/nigiri/loader/gtfs/route.h

```diff
@@ -28,6 +28,8 @@ struct route {
 
 using route_map_t = hash_map<std::string, std::unique_ptr<route>>;
 
+clasz to_clasz(std::uint16_t);
```

Comment:
in the global context, maybe `route_type_to_class` gives more information about the parameter than `std::uint16_t`